src/FooterLinkManagerInterface.php

```diff
@@ -44,4 +44,15 @@ public function getLinksWithoutSection(): array;
    */
   public function getSectionsAsOptions(): array;
 
+  /**
+   * The list cache tags associated with this entity type.
+   *
+   * Enables code listing entities of this type to ensure that newly created
+   * entities show up immediately.
+   *
+   * @return array
+   *   The list cache tags.
+   */
+  public function getListCacheTags(): array;
```

Comment:
Let's remove this form here, it's not this service's scope to return cache tags of two entity type storage.